CanvasPlusPlayground/Features/Announcements/CourseAnnouncementsView.swift

```diff
@@ -52,3 +41,82 @@ struct CourseAnnouncementsView: View {
     }
 }
 
+private struct AnnouncementRow: View {
+    let course: Course
+    let announcement: Announcement
+
+    var body: some View {
+        VStack(alignment: .announcementRowAlignment) {
+            header
+            detail
+        }
+    }
+
+    private var header: some View {
+        HStack {
+            Group {
+                if !(announcement.isRead ?? false) {
+                    Circle()
+                        .fill(.tint)
+                } else {
+                    Spacer().frame(width: unreadIndicatorWidth)
```

Comment:
is the frame here redundant?

Reply:
true, leftover from a prev version

Reply:
nvm, it needs to be there for some reason